specs/swagger.yml

```diff
@@ -4802,3 +5051,80 @@ definitions:
 
   specBodyDetail:
     type: string
+
+  listRoleAttributesEnvelope:
+    type: object
+    properties:
+      meta:
+        $ref: '#/definitions/meta'
+      data:
+        $ref: '#/definitions/roleAttributesList'
+
+  roleAttributesList:
+    description: An array of role attributes
+    type: array
+    items:
+      type: string
+
+  getIdentityPolicyAdviceEnvelope:
+    type: object
+    properties:
+      meta:
+        $ref: '#/definitions/meta'
+      data:
+        $ref: '#/definitions/policyAdvice'
+
+  policyAdvice:
+    type: "object"
+    required:
+      - "identityId"
+      - "identity"
+      - "serviceId"
+      - "service"
+      - "isBindAllowed"
+      - "isDialAllowed"
+      - "identityRouterCount"
+      - "serviceRouterCount"
+      - "commonRouters"
+    properties:
```

Comment:
`required` section not necessary as this is a read object.

Reply:
Removed